engine/flink/model-util/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/aggregate/sampleTransformers.scala

```diff
@@ -32,14 +32,14 @@ object sampleTransformers {
                 @DualEditor(simpleEditor = new SimpleEditor(
                   `type` = SimpleEditorType.FIXED_VALUES_EDITOR,
                   possibleValues = Array(
-                    new LabeledExpression(label = "First", expression = "T(pl.touk.nussknacker.engine.flink.util.transformer.aggregate.AggregateHelper).FIRST"),
-                    new LabeledExpression(label = "Last",  expression = "T(pl.touk.nussknacker.engine.flink.util.transformer.aggregate.AggregateHelper).LAST"),
-                    new LabeledExpression(label = "Min",   expression = "T(pl.touk.nussknacker.engine.flink.util.transformer.aggregate.AggregateHelper).MIN"),
-                    new LabeledExpression(label = "Max",   expression = "T(pl.touk.nussknacker.engine.flink.util.transformer.aggregate.AggregateHelper).MAX"),
-                    new LabeledExpression(label = "Sum",   expression = "T(pl.touk.nussknacker.engine.flink.util.transformer.aggregate.AggregateHelper).SUM"),
-                    new LabeledExpression(label = "List",  expression = "T(pl.touk.nussknacker.engine.flink.util.transformer.aggregate.AggregateHelper).LIST"),
-                    new LabeledExpression(label = "Set",   expression = "T(pl.touk.nussknacker.engine.flink.util.transformer.aggregate.AggregateHelper).SET"),
-                    new LabeledExpression(label = "ApproximateSetCardinality", expression = "T(pl.touk.nussknacker.engine.flink.util.transformer.aggregate.AggregateHelper).APPROX_CARDINALITY")
+                    new LabeledExpression(label = "First", expression = "#AGG.first"),
```

Comment:
I have kind of mixed feelings about this "#AGG"... we assume that there *is* helper #AGG - which is part of generic model, not of model-util... Let's at least add some comment on that?

Reply:
I've added documentation.